lib/LanguagePicker.js

```diff
@@ -76,43 +76,48 @@ LanguagePicker.prototype.newProcessor = function newProcessor() {
       }
 
       // 2. Fallback language from fallbacks.json
-      for (let i = 0; i < this.fallbacks.length; i++) {
-        if (this.values[this.fallbacks[i]]) {
-          return this.values[this.fallbacks[i]];
+      for (const fallback of this.fallbacks) {
+        val = this.values[fallback];
+        if (val) {
+          return val;
         }
       }
 
       // 3. Any language with suffix of same script as requested language
-      for (let i = 0; i < valueLangCodes.length; i++) {
-        if (valueLangCodes[i].endsWith(this.langScript)) {
-          return this.values[valueLangCodes[i]];
+      const valueLangCodes = Object.keys(this.values);
+      for (const langCode of valueLangCodes) {
+        val = this.values[langCode];
```

Comment:
why do you need to do a lookup and check it, if you know that every langCode must exist in `this.values` (because you just created it that way)?  I would rewrite:
```js
 for (const langCode of valueLangCodes) {
  if (langCode.endsWith(this.langScript)) {
    return this.values[langCode];
  }
}
```